syncs/syncs_test.go

```diff
@@ -137,4 +137,23 @@ func TestMap(t *testing.T) {
 			t.Errorf("exactly one LoadOrStore should load")
 		}
 	})
+
+	t.Run("Clear", func(t *testing.T) {
+		var m Map[string, string]
+		_, _ = m.LoadOrStore("a", "1")
+		_, _ = m.LoadOrStore("b", "2")
+		_, _ = m.LoadOrStore("c", "3")
+		_, _ = m.LoadOrStore("d", "4")
+		_, _ = m.LoadOrStore("e", "5")
+
+		if m.Len() != 5 {
+			t.Errorf("Len after loading want=5 got=%d", m.Len())
+		}
+
+		m.Clear()
+		if m.Len() != 0 {
+			t.Errorf("Len after Clear want=0 got=%d", m.Len())
+		}
+	})
+
```

Comment:
Trailing newline

Reply:
Done. However lint is still unhappy, doesn't say why, and doesn't run locally.

```
dgentry@ubuntu:tailscale/tailscale/syncs$ golangci-lint run --new=false --new-from-rev=
ERRO Running error: unknown linters: 'bidichk,revive', run 'golangci-lint linters' to see the list of supported linters
```

Reply:
Done.